lib/components/Datepicker/Datepicker.tsx

```diff
@@ -85,19 +90,19 @@ export const Datepicker = React.forwardRef<HTMLInputElement, DatepickerProps>(fu
             }}
             onBlur={onBlur}
           />
-          <ArkDatePicker.Trigger className="material-symbols-outlined select-none rounded-r-[10px] border-y-[5px] border-r-[5px] border-border-gray bg-white p-[11px] text-secondary-gray">
+          <ArkDatePicker.Trigger className="material-symbols-outlined select-none rounded-r border-y border-r border-border-gray bg-white p-5 text-secondary-gray">
             calendar_month
           </ArkDatePicker.Trigger>
         </div>
       </ArkDatePicker.Control>
       {help && (
-        <div id={helpId} className="mt-2 block text-help text-secondary-3">
+        <div id={helpId} className="mt-2 block text-help text-secondary-gray">
           {help}
         </div>
       )}
       <Portal>
         <ArkDatePicker.Positioner>
-          <ArkDatePicker.Content className="z-50 rounded-[20px] border-[3px] border-secondary-gray bg-white p-4">
+          <ArkDatePicker.Content className="z-50 rounded border border-secondary-gray bg-white p-4">
```

Comment:
In the new design, there is no border. It is done with the `box-shadow`

Reply:
Changed. I created a config for it, in case we need more similar borders with box-shadow.